src/ser/mod.rs

```diff
@@ -112,6 +96,8 @@ pub struct PrettyConfig {
     /// Enumerate array items in comments
     #[serde(default = "default_enumerate_arrays")]
     pub enumerate_arrays: bool,
+    /// Enable implicit_some extension
+    pub implicit_some: bool,
```

Comment:
We would potentially want other features there as well. Let's make it a bitflags field to account for future needs?
We could move the `struct_names` in these flags too, sometimes later.

Reply:
Sure. I'm going to be busy for the next week, but I'll try to get to it after that.